browser/src/layer/tile/TileLayer.TableOverlay.js

```diff
@@ -324,6 +333,57 @@ L.CanvasTileLayer.include({
 		e.target.dragging.freezeY(true);
 	},
 
+	_addInsertMarkers: function(type, start) {
+		const markersDistanceFromTable = 10;
+		const delta1 = this._convertPixelToTwips(markersDistanceFromTable);
+		
+		let documentPosition;
+		let markerWidth;
+		let markerHeight;
+		
+		if (type === 'column') {
+			const tableRightEdge = parseInt(this._currentTableData.columns.right);
+			const columnPosition = this._tablePositionColumnOffset + tableRightEdge + delta1;
+			documentPosition = { pX: columnPosition, pY: start };
+
+			// Total column height and width for insert column marker
+			const rowStart = this._tablePositionRowOffset + parseInt(this._currentTableData.rows.left);
+			const rowEnd = this._tablePositionRowOffset + parseInt(this._currentTableData.rows.right);
+			const totalHeightTwips = rowEnd - rowStart;
+			markerHeight = this._convertTwipsToPixels(new L.Point(0, totalHeightTwips)).y;
+			markerWidth = 16;
+
+		} else {
+			const tableBottomEdge = parseInt(this._currentTableData.rows.right);
+			const rowPosition = this._tablePositionRowOffset + tableBottomEdge + delta1;
+			documentPosition = { pX: start, pY: rowPosition };
+
+			// Total row width and height for insert row marker
+			const colStart = this._tablePositionColumnOffset + parseInt(this._currentTableData.columns.left);
+			const colEnd = this._tablePositionColumnOffset + parseInt(this._currentTableData.columns.right);
+			const totalWidthTwips = colEnd - colStart;
+			markerWidth = this._convertTwipsToPixels(new L.Point(totalWidthTwips, 0)).x;
+			markerHeight = 16;
+		}
+
+		const clickHandler = () => {
+			if (type === 'column') {
+				app.socket.sendMessage('uno .uno:InsertColumnsAfter');
+			} else {
+				app.socket.sendMessage('uno .uno:InsertRowsAfter');
+			}
+		};
+
+		const markerSection = new app.definitions.tableInsertMarkerSection(type, documentPosition, clickHandler);
+		markerSection.setMarkerSize(markerWidth, markerHeight);
```

Comment:
This marker size can be a local hard-coded variable inside TableInsertMarkerSection.
The more we move this new-section related code into the new section's file the better.

Reply:
@gokaysatir the marker size can't be hard-coded since it depends on the width and height of the table

@pedropintosilva wrote 
>For the buttons I think it would be better to have the width and height dynamically (following the row's width or column's height). This way the distance that user needs to do with his mouse pointer will be shorter and less aiming problems.
>

Reply:
Sorry i meant MarkerWidth and MarkerHeight. Those variables with value of 16 etc.

Reply:
@gokaysatir but both row and column marker have different heights and widths for example 
`markerWidth` for `row marker` is set using table width..
`markerheight for `column marker` is set using table height

Reply:
Hmm, okay thanks for the comment :)